### PR TITLE
docs - fix quick typo in filter documentation

### PR DESCRIPTION
**Pull request by briantist**
fix a small typo introduced in #75343

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by bmillemathias — COMMENTED** [review]
ok for me, while I don't like this kind of formulation as this is a bit vague.

**Comment by Akasurde** [comment]
@bmillemathias Can you explain a bit more?

File: docs/docsite/rst/user_guide/playbooks_filters.rst [hunk 1]
```diff
@@ -1522,7 +1522,7 @@ The ``regex_search`` filter returns an empty string if it cannot find a match::
     {{ 'ansible' | regex_search('foobar') }}
     # => ''
 
-Note that due to historic behavior and custom re-implementation of some of the Jinja internals in Ansible there is an exception to the behavior. When used in a Jinja expression (for example in conjunction with operators, other filters, and so no) the return value differs, in those situations the return value is ``none``. See the two examples below::
+Note that due to historic behavior and custom re-implementation of some of the Jinja internals in Ansible there is an exception to the behavior. When used in a Jinja expression (for example in conjunction with operators, other filters, and so on) the return value differs, in those situations the return value is ``none``. See the two examples below::
```
Comment (Akasurde) [inline docs/docsite/rst/user_guide/playbooks_filters.rst:1525]: @acozine @samccann Can you please help us with word-smithing?

Comment (Akasurde) [reply docs/docsite/rst/user_guide/playbooks_filters.rst:1525]: @briantist Typo correction is OK. I too feel that this sentence needs improvement.

**Comment by briantist** [comment]
I'm not the original author, this is just a 1 character typo fix. I think if you all want to update the wording a new PR is probably best, but if something is quickly agreed upon and added as a suggestion I'll apply it.

**Comment by Akasurde** [comment]
> I'm not the original author, this is just a 1 character typo fix. I think if you all want to update the wording a new PR is probably best, but if something is quickly agreed upon and added as a suggestion I'll apply it.

Yup. Let us wait for the Docs team to suggest something.

**Comment by samccann** [comment]
@briantist @Akasurde @bmillemathias - Let's merge this as is. The rewording affects  a short section after this that isn't part of this PR. I'll open that new PR asap for y'all to review.

**Review by samccann — APPROVED** [review]
opening a separate PR to wordsmith this a bit.

**Pull request merged**